v2/pkg/subscraping/sources/hackertarget/hackertarget_test.go

```diff
@@ -0,0 +1,67 @@
+package hackertarget_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/projectdiscovery/subfinder/v2/pkg/subscraping"
+	"github.com/projectdiscovery/subfinder/v2/pkg/subscraping/sources/hackertarget"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHackerTargetWithActualDomain(t *testing.T) {
+	//given
+	ctx := context.Background()
+	domain := "projectdiscovery.io"
+	keys := &subscraping.Keys{}
+	session, err := subscraping.NewSession(domain, keys, "", 0, 10)
+	if err != nil {
+		t.Fatalf("Expected nil got %v while creating session\n", err)
+	}
+
+	hackertarget := hackertarget.Source{}
+
+	//when
+	result := <-hackertarget.Run(ctx, domain, session)
+
+	//expect
+	assert.NotNil(t, result.Value)
```

Comment:
We should do some stricter assertions.

Reply:
@forgedhallpass Did some changes; please re-check. Since this is the first isolated integration test, I want it to be good. We can discuss the details. Let me know what you think.